candidate-standard/clause_0_front_material.adoc

```diff
@@ -2,9 +2,9 @@
 
 [big]*i.     Abstract*
 
-The Environmental Data Retrieval (EDR) Application Programming Interface (API) provides a family of lightweight query interfaces to access Environmental Data resources by requesting data at a *Position*, within an *Area* or along a *Trajectory*. An Environmental Data resource is a collection of spatiotemporal data that can be sampled using the EDR query pattern geometries. These patterns are described in the <<rc_core-section,Requirement Class Core>> section.
+The Environmental Data Retrieval (EDR) Application Programming Interface (API) provides a family of lightweight query interfaces to access spatio-temporal data resources by requesting data at a *Position*, within an *Area* or along a *Trajectory*. A spatio-temporal data resource is a collection of spatio-temporal data that can be sampled using the EDR query pattern geometries. These patterns are described in the <<rc_core-section,Requirement Class Core>> section.
```

Comment:
Global comment: should spatio-temporal be spatiotemporal? https://en.wikipedia.org/wiki/Spatiotemporal_database

Reply:
@tomkralidis Just had discussion with Jeremy Tandy and @m-burgoyne. Spatiotemporal may be slightly better than spatio-temporal. Acronym becomes SDR rather than STDR.